src/pre_defined_classes/simple_programs_taking_numeric_user_input/hard/q6/ZipcodeValidator.java

```diff
@@ -0,0 +1,45 @@
+package pre_defined_classes.simple_programs_taking_numeric_user_input.hard.q6;
+
+import java.util.Scanner;
+
+/*
+An online e-commerce website uses zip codes to determine the rate of the shipment. Prior to doing so, it validates the
+zip code by asking the user to re-submit the same zip code and double confirm the zip-code. Make a program that asks
+the user to enter a zip code and confirm it, then display out both zip codes back to the user.
+Note that American zip codes consist of 5 digits.
+
+
+Sample Output:
+    Enter a US zip code:
+    12345
+    Re-enter the US zip code:
+    54321
+    The first zip code is: 12345, and the second zip is: 54321
+
+    Enter a US zip code:
+    123
+    Re-enter the US zip code:
+    123
+    The first zip code is: 123, and the second zip is: 123
+
+    Enter a US zip code:
+    01234
+    Re-enter the US zip code:
+    01234
+    The first zip code is: 1234, and the second zip is: 1234
+    //the data type of the input should be String type instead
+    >> okay changed to .nextLine(). Actually, nah, if I changed to a string question, this question should be under pre_defined_classes-simple_programs_taking_string_user_input
+    >> I changed ZipcodeValidator.java & MainTest.java (take string input). The google doc is still takes int input just in case.
```

Comment:
As this is programs taking numeric input, it should stay as int input, and then in the question text just mention something along the lines that 0 is not valid for the zip codes.

Reply:
That way we avoid this error

Reply:
That would be a great way to avoid error, but we then have to change the question so it's not based on the US zip code because some of the zip code in the US does start with zero. It would be confusing to students who knows the US zip code or Google while doing the question.
![image](https://user-images.githubusercontent.com/60047943/177417080-21c9fd72-5907-481a-a80f-d70deb735d8f.png)

Reply:
We could just say assume the zip code will not start with 0